Author: <joerg@dorchain.net>
Description: Replace all occurences of local in pathnames
Index: lcr-1.6~20090906/Makefile.am
===================================================================
--- lcr-1.6~20090906.orig/Makefile.am	2009-09-07 13:35:03.834457538 +0200
+++ lcr-1.6~20090906/Makefile.am	2009-09-07 13:36:12.292950699 +0200
@@ -20,19 +20,19 @@
 ##    the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
 ##    Boston, MA 02110-1301, USA.
 
-INSTALLdir = $(DESTDIR)/usr/local/lcr
+INSTALLdir = $(DESTDIR)/usr/share/lcr
 
 pkglogdir=$(localstatedir)/log/@PACKAGE@
-pkgsysconfdir=$(sysconfdir)/@PACKAGE@
+pkgsysconfdir=/etc/@PACKAGE@
 
-#CONFIGdir=$(DESTDIR)$(pkgsysconfdir)
+CONFIGdir=$(DESTDIR)$(pkgsysconfdir)
 #SHAREdir=$(DESTDIR)$(pkgdatadir)
-#LOGdir=$(DESTDIR)$(pkglogdir)
+LOGdir=$(DESTDIR)$(pkglogdir)
 #EXTENSIONdir=$(DESTDIR)$(localstatedir)/lib/lcr
 
-CONFIGdir=$(INSTALLdir)
+#CONFIGdir=$(INSTALLdir)
 SHAREdir=$(INSTALLdir)
-LOGdir=$(INSTALLdir)
+#LOGdir=$(INSTALLdir)
 EXTENSIONdir=$(INSTALLdir)/extensions
 
 astmoddir = $(DESTDIR)/usr/lib/asterisk/modules
Index: lcr-1.6~20090906/genrc.c
===================================================================
--- lcr-1.6~20090906.orig/genrc.c	2009-09-07 13:35:03.878459355 +0200
+++ lcr-1.6~20090906/genrc.c	2009-09-07 13:36:12.292950699 +0200
@@ -80,12 +80,12 @@
 
 	n = i;
 
-	printf("\nWhere do you like to load the modules from, enter 0 for default, 1 for\n'/usr/local/lcr/modules/' or the full path.\n[0 | 1 | <path>]: "); fflush(stdout);
+	printf("\nWhere do you like to load the modules from, enter 0 for default, 1 for\n'/usr/lib/lcr/modules/' or the full path.\n[0 | 1 | <path>]: "); fflush(stdout);
 	scanf("%s", input);
 	if (!strcmp(input, "0"))
 		SCPY(input, "");
 	if (!strcmp(input, "1"))
-		SCPY(input, "/usr/local/lcr/modules");
+		SCPY(input, "/usr/lib/lcr/modules");
 	if (input[0]) if (input[strlen(input)-1] != '/')
 		SCAT(input, "/");
 
Index: lcr-1.6~20090906/options.c
===================================================================
--- lcr-1.6~20090906.orig/options.c	2009-09-07 13:35:03.898459331 +0200
+++ lcr-1.6~20090906/options.c	2009-09-07 13:36:12.296951419 +0200
@@ -19,7 +19,7 @@
 #include "options.h"
 
 struct options options = {
-	"/usr/local/lcr/log",		/* log file */
+	"/var/log/lcr/log",		/* log file */
 	0x0000,				/* debug mode */
 	'a',				/* a-law */
 	"0",				/* national prefix */
Index: lcr-1.6~20090906/default/options.conf
===================================================================
--- lcr-1.6~20090906.orig/default/options.conf	2009-09-07 13:35:03.858458743 +0200
+++ lcr-1.6~20090906/default/options.conf	2009-09-07 13:36:12.296951419 +0200
@@ -24,8 +24,8 @@
 
 # The log file can be used to track actions by the LCR. Omit the parameter
 # to turn off log file. By default, log file is located inside the directory
-# "/usr/local/lcr/log".
-#log /usr/local/lcr/log
+# "/var/log/lcr/log".
+#log /var/log/lcr/log
 
 # Use "alaw" (default) or "ulaw" samples.
 #alaw
